Vizzuality/heco-invest
# Let 174 project developer form

### Add the project developer setup page
Create page with form using multi-page layout
Add validations
Add service

## Testing instructions

* Respects the [visual design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=70%3A14667)

* Contains all the fields defined in the “Forms” tab of the [requirements document](https://docs.google.com/spreadsheets/d/1qtP-qkm8iS6OlluiJwRvQoi6rCns9iKhRYoNzc9qkGg/edit#gid=1884416048)

* The fields are validated:

  * required fields are not empty

* Errors are displayed if:

  * the validation does not pass

  * the API doesn't respond with success

* Accessible by [WCAG 2.1 standards](https://www.w3.org/TR/WCAG21/)

* The fields are part of a form

* Input and form errors are correctly presented to assistive technologies (AT) ([see this guide on form validation](https://webaim.org/techniques/formvalidation/))

* The correct locale is set on the inputs ([see documentation of the lang attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)) if it’s different from the page’s

* Available for translation in Spanish, Portuguese and English

* Page works well on desktop devices

## Tracking

[LET-174](https://vizzuality.atlassian.net/browse/LET-174)
